src/Symfony/Component/Serializer/Normalizer/DataUriNormalizer.php

```diff
@@ -46,7 +46,7 @@ public function __construct(MimeTypeGuesserInterface $mimeTypeGuesser = null)
 
     public function getSupportedTypes(?string $format): array
     {
-        $isCacheable = __CLASS__ === static::class || $this->hasCacheableSupportsMethod();
+        $isCacheable = true;
 
         return [
             \SplFileInfo::class => $isCacheable,
```

Comment:
```suggestion
        return [
            \SplFileInfo::class => true,
```

and the next lines also of course